android-buddy-plugin/src/main/java/com/likethesalad/android/buddy/extension/libraries/TransformationScopeExtension.kt

```diff
@@ -0,0 +1,17 @@
+package com.likethesalad.android.buddy.extension.libraries
+
+
+import com.likethesalad.android.buddy.configuration.TransformationScopeType
+import org.gradle.api.model.ObjectFactory
+import javax.inject.Inject
+
+@Suppress("UnstableApiUsage")
+open class TransformationScopeExtension @Inject constructor(objectFactory: ObjectFactory) {
+    val scope = objectFactory.property(String::class.java)
```

Comment:
It's probably worth keeping the set of strings, because right now it looks quite strange for a user to set "DEPENDENCIES" as scope and then expect that said scope would support both the project and dependencies' transformations.

Also probably some users would want to only transform dependencies, and with this option it seems like they won't be able to do so 😅

Another option would be adding a 3rd item to the enum that express this case better, such as "ALL", and then leave the "DEPENDENCIES" one to only transform dependencies. I personally like this option the better.

Reply:
Actually, if you haven't made any changes yet, I think it's better just to rename the enum item "DEPENDENCIES" to "ALL" in order to describe better what's already in the code. Then later in the future, if somebody needs to be able of only transforming dependencies, I can add a new item to the enum then.

After the rename I'd say we're ready to merge 👍

Reply:
👍 renamed as proposed!

Reply:
Also, once merged, would you mind creating a new release and uploading it to maven central so I could use that version instead of a local one? 

Thanks again!

Reply:
Cheers! yeah, I was planning to release a new version today in order to sort this bug: #4 - But now I'll also add your changes as part of it as well 👍 

Thanks!